.github/workflows/ci.yaml

```diff
@@ -143,6 +143,15 @@ jobs:
       - name: Create k8s Kind Cluster
         run: make kind-up
 
+      - name: Run unit tests
+        run: make test
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@f30e4959ba63075080d4f7f90cacc18d9f3fafd7 # v4.0.0
+        with:
+          fail_ci_if_error: true
```

Comment:
Why don't you test this field using Codecov 3.1.4, checking reports work, merging Renovate v4.0.0 and seeing the error reported in another PR?

Tokenized reports should still work in v3